### PR TITLE
radarr: 0.2.0.1120 -> 0.2.0.1217

### PR DESCRIPTION
**Pull request by nyanloutre**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/radarr/default.nix [hunk 1]
```diff
@@ -2,11 +2,11 @@
...
+  version = "0.2.0.1217";
 
   src = fetchurl {
-    url = "https://github.com/Radarr/Radarr/releases/download/v${version}/Radarr.develop.${version}.linux.tar.gz";
-    sha256 = "0vsjaza0k4djb3vnazl2py4qmbxqfyyr0x9p4flq78yn79hp3439";
+    url = "https://github.com/Radarr/Radarr/releases/download/v${version}/Radarr.v${version}.linux.tar.gz";
+    sha256 = "09zzvfqpv58b79a906013pjq42qwbibf16rz24gnqg7wq7az83jy";
   };
 
   buildInputs = [ makeWrapper ];
```
Comment (c0bw3b) [inline pkgs/servers/radarr/default.nix]:
```suggestion
  nativeBuildInputs = [ makeWrapper ];
```

**Comment by c0bw3b** [comment]
(nitpick) and short description should not contain a trailing dot

@GrahamcOfBorg build radarr

**Comment by nyanloutre** [comment]
@c0bw3b I applied your suggestions :+1:

**Comment by worldofpeace** [comment]
@nyanloutre Merged in 43dd779a8d019a0667e3eed1346f83ffe2004e05

Thanks for the update :sparkle:

**Pull request closed without merging**